### PR TITLE
Add add-sso-for-disqus-on-your-wordpress-site-with-auth0

### PR DESCRIPTION
**Pull request by mostekcm**
@kukicado here it is, sorry it took so long to create the PR!  Let me know if I need to change anything.

### CODE REVIEW

File: _posts/2017-03-22-add-sso-for-disqus-on-your-wordpress-site-with-auth0.markdown [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+  - WordPress
+  - SSO
+  - Identity
+related:
+  - 2014-07-02-wordpress-single-sign-on
+---
+
+---
+
+**TL;DR** [Disqus](https://disqus.com/) is a great tool for adding interactivity through conversation and comments to your blog.  Integrating it with your WordPress site is really simple. The downside to Disqus is that it requires another set of credentials before users can interact with your site. Today, we'll show how to eliminate the need for users to login twice with Auth0 and Single Sign On (SSO) for a better user experience.
```
Comment (kmaida) [inline _posts/2017-03-22-add-sso-for-disqus-on-your-wordpress-site-with-auth0.markdown]: `login` -> "log in"

(Used here as a verb, not a noun.)

File: _posts/2017-03-22-add-sso-for-disqus-on-your-wordpress-site-with-auth0.markdown [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+[WordPress](https://wordpress.com/) makes it easy to create blogs and other media sites. It is one of the leading CMS systems out there, and allows developers to create powerful websites that users can engage with. The default comment capability from WordPress can be greatly improved through the addition of the [Disqus WordPress Plugin](https://wordpress.org/plugins/disqus-comment-system/). Disqus provides a really nice interface for tracking and interacting with users through comments and reviews.  The one downfall to Disqus is that it requires users to have a separate login alongside their WordPress credentials. That’s where SSO comes in. With SSO enabled for your Disqus account, your WordPress user accounts will automatically integrate with Disqus, enabling users to comment without having to log into Disqus as well as WordPress.
+
+---
+
+## Configuring Auth0 for Wordpress
+There’s already plenty of good information for how to setup the WordPress plugin for Auth0.  Simply follow [these steps](https://auth0.com/docs/cms) and you'll have Auth0 integrated in no time.
```
Comment (kmaida) [inline _posts/2017-03-22-add-sso-for-disqus-on-your-wordpress-site-with-auth0.markdown]: `how to setup the` -> "set up" (two words; used as a verb, not a noun in this context)

File: _posts/2017-03-22-add-sso-for-disqus-on-your-wordpress-site-with-auth0.markdown [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+Make sure you get to the point where the [Lock](https://www.google.com/url?q=https://auth0.com/lock&sa=D&ust=1487712650254000&usg=AFQjCNGgfaHnBtRJcZNxH4tVKkaV_vyeVA) widget is displayed when attempting to login:
```
Comment (kmaida) [inline _posts/2017-03-22-add-sso-for-disqus-on-your-wordpress-site-with-auth0.markdown]: `login` -> "log in" (verb)

File: _posts/2017-03-22-add-sso-for-disqus-on-your-wordpress-site-with-auth0.markdown [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+
+![Login Widget](https://cdn.auth0.com/blog/auth0-disqus-wordpress-sso/login-widget.png)
+
+## Adding Disqus
+To utilize Disqus, you need a Disqus account.  Signing up for an account will enable you to configure your applications to use Disqus. We’ll enable SSO for this primary account. Note, you can only configure one SSO domain, so if you have multiple sites, you may have to create a special account for each site.  I used my Auth0 email and set a password.  Then created this demo organization:
```
Comment (kmaida) [inline _posts/2017-03-22-add-sso-for-disqus-on-your-wordpress-site-with-auth0.markdown]: `Then created this demo organization:` -> "I then created..." (Change into a more complete sentence.)

File: _posts/2017-03-22-add-sso-for-disqus-on-your-wordpress-site-with-auth0.markdown [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+
+![Create New Site Form](https://cdn.auth0.com/blog/auth0-disqus-wordpress-sso/create-new-site.png)
```
Comment (kmaida) [inline _posts/2017-03-22-add-sso-for-disqus-on-your-wordpress-site-with-auth0.markdown]: You may want to add a more specific alt tag to this image, possibly mentioning WordPress and Disqus. This helps accessibility and SEO.

File: _posts/2017-03-22-add-sso-for-disqus-on-your-wordpress-site-with-auth0.markdown [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+
+![Disqus Comment System Plugin](https://cdn.auth0.com/blog/auth0-disqus-wordpress-sso/disqus-comment-system-plugin.png)
+
+Once activated, let’s finish setting up Disqus:
+
+1. Go to the “Settings” for Disqus under “Comments->Disqus” in your admin section of WordPress. 
+1. Click Upgrade
+1. Enter your Disqus account credentials.
+1. Choose the website that you configured earlier
+
```
Comment (kmaida) [inline _posts/2017-03-22-add-sso-for-disqus-on-your-wordpress-site-with-auth0.markdown]: Some of the above steps have periods and some don't. Please make them all consistent.

File: _posts/2017-03-22-add-sso-for-disqus-on-your-wordpress-site-with-auth0.markdown [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+At this point you should be able to log into your site. Go to a blog post, and see the default comment section replaced with Disqus.  Notice that you have to log in to Disqus to be able to comment (if you don’t have to log in, you were probably already logged into Disqus, try logging out using the dropdown).
```
Comment (kmaida) [inline _posts/2017-03-22-add-sso-for-disqus-on-your-wordpress-site-with-auth0.markdown]: `have to log in to Disqus` -> "have to log into Disqus"

File: _posts/2017-03-22-add-sso-for-disqus-on-your-wordpress-site-with-auth0.markdown [hunk 8]
```diff
@@ -0,0 +1,118 @@
...
+```text
+I have a WordPress site that I am integrating with Disqus and I would like to have SSO enabled for that site so users don’t also have to put in their Disqus password.
+
+Thanks,
+Carlos
+```
+
+![Request SSO Form](https://cdn.auth0.com/blog/auth0-disqus-wordpress-sso/request-sso.png)
+￼
+Unfortunately, you will have to wait 24-48 hours until they enable SSO functionality for your account. While you wait, they will send you some additional details on how to setup SSO. There is a lot of good reading here and I suggest you check it out. We will use the SSO on [WordPress example](https://help.disqus.com/customer/portal/articles/1148635-setting-up-sso-on-wordpress).
```
Comment (kmaida) [inline _posts/2017-03-22-add-sso-for-disqus-on-your-wordpress-site-with-auth0.markdown]: `setup` -> "set up" (verb)

File: _posts/2017-03-22-add-sso-for-disqus-on-your-wordpress-site-with-auth0.markdown [hunk 9]
```diff
@@ -0,0 +1,118 @@
...
+
+## Configuring SSO
+Once you get the email that SSO has been enabled, you can continue this tutorial. We’re only a few steps away from having SSO functionality up and running. Let’s get to it.
+
+### Configure the SSO domain
+Go to [the SSO configuration page](https://disqus.com/api/sso/):
+
+![SSO Config](https://cdn.auth0.com/blog/auth0-disqus-wordpress-sso/sso-config.png)
+
+Here we will configure our SSO application. The information will be dependent on how you setup your application. In my case:
```
Comment (kmaida) [inline _posts/2017-03-22-add-sso-for-disqus-on-your-wordpress-site-with-auth0.markdown]: `setup` -> "set up" (verb)

File: _posts/2017-03-22-add-sso-for-disqus-on-your-wordpress-site-with-auth0.markdown [hunk 10]
```diff
@@ -0,0 +1,118 @@
...
+
+Enter Name: Disqus SSO Demo
+Slug: mostekcm-disqus-sso-demo
+
+### Configure your application
+Got to [the registration page](http://disqus.com/api/applications/register/):
```
Comment (kmaida) [inline _posts/2017-03-22-add-sso-for-disqus-on-your-wordpress-site-with-auth0.markdown]: `Got to` -> "Go to"

File: _posts/2017-03-22-add-sso-for-disqus-on-your-wordpress-site-with-auth0.markdown [hunk 11]
```diff
@@ -0,0 +1,118 @@
...
+
+![Application Registration](https://cdn.auth0.com/blog/auth0-disqus-wordpress-sso/application-config.png)
+
+### Grab your API keys
+On this page, copy your Secret and Public Keys:
+
+![Disqus Keys](https://cdn.auth0.com/blog/auth0-disqus-wordpress-sso/disqus-keys.png)
+
+### Configure Wordpress
+Back in wordpress, go to Comments->Disqus->Plugin Settings (far right tab).
```
Comment (kmaida) [inline _posts/2017-03-22-add-sso-for-disqus-on-your-wordpress-site-with-auth0.markdown]: Capitalize `wordpress`.

File: _posts/2017-03-22-add-sso-for-disqus-on-your-wordpress-site-with-auth0.markdown [hunk 12]
```diff
@@ -0,0 +1,118 @@
...
+Scroll down to the advanced section and paste the secret and public keys:
+
+![Wordpress Plugin SSO Config](https://cdn.auth0.com/blog/auth0-disqus-wordpress-sso/wordpress-plugin-sso-config.png)
+
+NOTE: Ensure that “Anyone can register” is set to true in General->Settings.
+
+## Give it a try!
+Once you log into wordpress, you should now see that you are logged into Disqus as well when you go to enter a comment:
```
Comment (kmaida) [inline _posts/2017-03-22-add-sso-for-disqus-on-your-wordpress-site-with-auth0.markdown]: Capitalize `wordpress`.

File: _posts/2017-03-22-add-sso-for-disqus-on-your-wordpress-site-with-auth0.markdown [hunk 13]
```diff
@@ -0,0 +1,118 @@
...
+
+![SSO is Working](https://cdn.auth0.com/blog/auth0-disqus-wordpress-sso/sso-works.png)
+
+That’s it! You know have SSO functionality for your WordPress sites. Users signing up and logging in will no longer be required to enter a second set of credentials to leave comments on your site.
```
Comment (kmaida) [inline _posts/2017-03-22-add-sso-for-disqus-on-your-wordpress-site-with-auth0.markdown]: `You know have` -> "You now have"

Sometimes the article references "WordPress", other times "Wordpress", and finally, "wordpress". Please go through the article and make sure all references say "WordPress". Be sure to check headings too.

File: _posts/2017-03-22-add-sso-for-disqus-on-your-wordpress-site-with-auth0.markdown [hunk 14]
```diff
@@ -0,0 +1,118 @@
...
+
+## Conclusion
+Today we saw how you can easily improve the user experience on your WordPress website with Disqus, Auth0, and Single Sign On (SSO). Lowering the barrier to entry for your users is always a plus, and not forcing them to enter another set of credentials to interact with your site is a big win here. If you are running WordPress applications and using Disqus for comments, Auth0 as we saw can save time implementing SSO. Give it a try today.
```
Comment (kmaida) [inline _posts/2017-03-22-add-sso-for-disqus-on-your-wordpress-site-with-auth0.markdown]: I would recommend removing `as we saw`.

**Comment by mostekcm** [comment]
Thank you @kmaida!  I have incorporated all of your changes.  I also found a couple more instances of login and setup.  🤦‍♂️ on the know...  Thanks for catching all of those things!

**Comment by kmaida** [comment]
Hi @mostekcm! You're welcome! The instances that weren't specifically cited in the comments were already correct though: or at least most of them were, it's possible I missed one or two, but several cases were right as they were. It might be helpful to go back through and change back the ones that weren't specifically called out.

(For instance, "log into" is grammatically correct. "Log in to" is less so.)

**Comment by mostekcm** [comment]
Thanks again @kmaida, fixed the "log in to".  Hopefully ready to merge now!  I did leave the only noun as login :).

**Pull request merged**